### ISSUE TITLE
pretibial dimple

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4767]
**Preferred term label:**
see simulconsult excel

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0001024

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I am adding this definition:
A groove or crease on the shins (pretibial, i.e., over the shin bone). Pretibial creases may be obvious at birth and may range from 3 cm to over 15 cm in length and lenghten as the limb grows. They appear as an elongated dimple because of the attachment of skin to underlying tissue (e.g., to the tibia). The dimple or crease grows in proportion to the growth of the leg.
PMID:23444103 

Added term as HP:0032538  @MickeySegal

**Comment by MickeySegal** [comment]
An example of this term is in https://www.ncbi.nlm.nih.gov/books/NBK1760/
